### ISSUE TITLE
PyTest Warning: Importing ABCs from collections deprecated

### ISSUE THREAD

**Issue by ZaydH** [fastai/fastai#2359]
**Describe the bug**

Running `pytest` on a module that imports `DataBunch` yields the warning:

```
<my_python>/site-packages/fastai/imports/core.py:9
  <my_python>/site-packages/fastai/imports/core.py:9: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import abc,  Counter, defaultdict, Iterable, namedtuple, OrderedDict

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Provide your installation details**
```text
=== Software ===
python       : 3.7.1
fastai       : 1.0.58
fastprogress : 0.1.21
torch        : 1.2.0
torch cuda   : None / is **Not available**

=== Hardware ===
No GPUs available

=== Environment ===
platform     : Darwin-18.7.0-x86_64-i386-64bit
conda env    : Unknown
python       : /Users/zayd/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/bin/python
sys.path     : /Users/zayd/repos/fastai_pytest_testcase
/Users/zayd/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/lib/python37.zip
/Users/zayd/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/lib/python3.7
/Users/zayd/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/lib/python3.7/lib-dynload
/Users/zayd/Library/Python/3.7/lib/python/site-packages
/Users/zayd/.pyenv/versions/3.7.1/Python.framework/Versions/3.7/lib/python3.7/site-packages
no supported gpus found on this system
```

`tox`: 3.14.0

**To Reproduce**

I created a [test case github repo](https://github.com/ZaydH/fastai_pytest_testcase).  Running the `tox` command in that directory should yield the warning.

**Expected behavior**

No warning in PyTest.

**Additional context**

Other large packages have ran into this issue.  If this is acknowledged as an issue, I can work to get you a PR quickly to address it.

**Comment by sgugger** [comment]
I'm guessing the fix is just to import the right abc. Is it compatible with python 3.6? That's the minimal version we require.

**Comment by ZaydH** [comment]
@sgugger I think the code is worth updating independent of Python version.  The `imports/core.py` module is double importing `abc`.  I do not see the second import of `abc` being used so I think it is unnecessary (please correct me if I am wrong).  

I will issue a pull request that changes the [line](https://github.com/fastai/fastai/blob/af395a701d6660543c49804b4f2cb53a6e67f261/fastai/imports/core.py#L9)

`from collections import abc,  Counter, defaultdict, Iterable, namedtuple, OrderedDict`

to:

```
from collections.abc import Iterable
from collections import Counter, defaultdict, namedtuple, OrderedDict
```

**Comment by sgugger** [comment]
Fixed by #2363
